include/openthread/border_agent.h

```diff
@@ -256,8 +256,11 @@ otError otBorderAgentSetEphemeralKey(otInstance *aInstance,
  *
  * @param[in] aInstance    The OpenThread instance.
  *
+ * @retval OT_ERROR_NONE           Successfully cleared the ephemeral key.
+ * @retval OT_ERROR_INVALID_STATE  Border Agent is connected to an external commissioner.
```

Comment:
What is the use-case for adding this error?

Reply:
Calling this method can lead to 2 different results: epskc is cleared or epskc is not cleared due to active connection. So for callers to understand if they have successfully cleared the epskc, currently they need to do an extra call to the `otBorderAgentIsEphemeralKeyActive`. Returning the error would make it a bit easier for the api users.